http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -444,7 +458,7 @@ class GKEInterpreter[F[_]](
               app.release,
               app.chart,
               dbCluster,
-              dbApp.nodepool.nodepoolName,
+              dbApp.nodepool.nodepoolName, // TODO: support autopilot mode
```

Comment:
Do you mean this is the TODO to support autopilot mode for Galaxy? Is there a JIRA ticket that could be linked?

Reply:
yea...I can create one

Reply:
created https://broadworkbench.atlassian.net/browse/IA-4987